packages/openworkflow/schema.ts

```diff
@@ -0,0 +1,140 @@
+/**
+ * Structural types for supported workflow input schemas. These mirror the
+ * public shapes from common validation libraries so we can infer types without
+ * depending on those packages directly.
+ *
+ * This file has been inspired by the `schemaTask` provided by trigger.dev.
+ * See reference file: https://github.com/triggerdotdev/trigger.dev/blob/main/packages/core/src/v3/types/schemas.ts
+ */
+
+/* eslint-disable @typescript-eslint/no-explicit-any, @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-return, @typescript-eslint/no-unsafe-call */
+
+export interface WorkflowSchemaZodLike<TInput, TParsed> {
```

Comment:
Can the `T` prefix be removed here in favor of `Input` and `ParsedInput` to match the generics naming style used in the rest of the codebase?